tests/test_serializers.py

```diff
@@ -116,11 +131,10 @@ def setUp(self, save_mock, manager_mock, get_queryset_mock):
         # This emulates property in the database, which needs to be updated
         # as a result of given test.
         self.existing_prop_mock = Mock(name='name', value='value')
-
-        filter_mock = Mock()
         manager_mock.filter = Mock(return_value=[self.existing_prop_mock])
 
         get_queryset_mock.return_value = Mock(get=Mock(return_value=1))
+        get_queryset_mock.__func__ = get_queryset_mock
```

Comment:
Because of this: https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/relations.py#L95